data_structures/binary_tree/binary_tree_mirror.py

```diff
@@ -0,0 +1,44 @@
+# Created by susmith98
+
+
+"""
+Problem Description:
+Given a binary tree, return it's mirror.
+"""
+
+
+def binaryTreeMirrorDict(
+    root: int, binary_tree: dict = {}, binary_tree_mirror: dict = {}
+):
+    if root is None or root not in binary_tree.keys():
+        return
+    left_child = binary_tree[root][0]
+    right_child = binary_tree[root][1]
+    binary_tree_mirror[root] = [right_child, left_child]
+    binaryTreeMirrorDict(left_child, binary_tree, binary_tree_mirror)
+    binaryTreeMirrorDict(right_child, binary_tree, binary_tree_mirror)
+
+
+def binaryTreeMirror(root: int, binary_tree: dict = {}) -> dict:
+    """
+    >>> binaryTreeMirror(1, { 1: [2,3], 2: [4,5], 3: [6,7], 7: [8,9]})
+    {1: [3, 2], 2: [5, 4], 3: [7, 6], 7: [9, 8]}
+    >>> binaryTreeMirror(5, { 1: [2,3], 2: [4,5], 3: [6,7], 7: [8,9]})
+    {}
+    >>> binaryTreeMirror\
```

Comment:
```suggestion
    >>> binaryTreeMirror(
```
Please format your code like psf/black would format it.  PEP8 suggests that backslashes should be avoided in Python code because any whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.